hypothesis/lms
# Assignment activity filters

> Depends on https://github.com/hypothesis/lms/pull/6473

This PR adds filters to the assignment section.

As it's been done in https://github.com/hypothesis/lms/pull/6473, the courses and assignments dropdowns start with the "active" course and assignment pre-selected. Then, selecting a different one navigates to that course or assignment and propagate the rest of the filters there.

This behavior might change in future, as the product decision is still under discussion. See https://github.com/hypothesis/lms/issues/6490

![Screenshot](https://github.com/user-attachments/assets/0f76a47b-b567-498a-8d82-e34446cd3b53)

### Testing steps

1. Go to the dashboards, and select one specific assignment.
2. Change selection in filters and verify it behaves as described above.

### Todo
